docs/collecting-data/collecting-from-own-applications/mobile-trackers/migration-guides/migration-guide-from-version-4-x-to-5-0/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "From version 4.x to 5.0"
+date: "2022-08-30"
```

Comment:
Not necessary for new pages (nor any page actually, but I have not removed it just to not touch the files)

Reply:
Removed for new pages